General/Baselines.md

```diff
@@ -771,20 +771,21 @@ The repository parameter is a string that can takes different form in case we ha
 
 The repository parameter to load a project from github/gitlab/bitbucket takes this form:
 
-`{prefix}://{owner}/{projectName}:{version}/{subFolder}`
+`{prefix}://{option}:{owner}/{projectName}:{version}/{subFolder}`
```

Comment:
```suggestion
`{prefix}://{optionalHostname}:{owner}/{projectName}:{version}/{subFolder}`
```

Reply:
I committed but since it is a server host url, why not "optionalHostURL" ?

Reply:
Because it is not a URL but only the hostname. See https://en.wikipedia.org/wiki/URL

Reply:
Ok then